examples/index.json

```diff
@@ -708,6 +708,20 @@
             }
         ]
     },
+    {
+        "title": "Interface to external code",
+        "column": 2,
+        "category": "Language concepts",
+        "samples": [
+            {
+                "name": "Interface to external code",
+                "url": "interface-to-external-code",
+                "verifyBuild": true,
+                "verifyOutput": true,
+                "isLearnByExample": true
+            }
```

Comment:
Does this work on the playground?

Reply:
Yes. Example `bal` file work on playground